.husky/pre-push

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+. "$(dirname "$0")/_/husky.sh"
+
+npx --no-install lint-staged
```

Comment:
Why pre-push? Why isn't pre-commit sufficient?

Reply:
[`pre-push`](https://git-scm.com/docs/githooks#_pre_push) provides another layer of validation ?

Reply:
But there is nothing that pre-push can catch that pre-commit doesn't? Is there? You can't push something that's not commited. Or am I missing something?

Reply:
Agreed, `pre-push` does nothing special than `pre-commit` but I personally use it everywhere, so it was a opinionated opinion I guess 😉 

- [x] Fixed in f1a4d5e